app/renderer/components/Stub.js

```diff
@@ -3,10 +3,10 @@ import React, { Component } from 'react';
 export default class Stub extends Component {
 
   render () {
-    const { someProp } = this.props;
+    const { someProp, t } = this.props;
 
     return <div>
-      Hello {someProp}!
+      {t('Hello')} {someProp}!
```

Comment:
the whole string should be a template

Reply:
This is just a stub for reference purpose. But I can change it anyways.